docs/examples/pipelinerun_details.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: steward.sap.com/v1alpha1
+kind: PipelineRun
+metadata:
+  generateName: ok-
```

Comment:
The name prefix should be identical to the name of this example file. This should be changed in all other example files, too.